vscode-dotnet-runtime-library/src/EventStream/LoggingObserver.ts

```diff
@@ -31,6 +31,6 @@ export class LoggingObserver implements ILoggingObserver {
     }
 
     private writeLine(line: string) {
-        this.log.concat(line);
+        this.log = this.log.concat(line);

```

Comment:
This is pretty subtle! Thank you for having such a concise fix and figuring it out! 😄